docs/tutorial/force_calibration.rst

```diff
@@ -212,3 +212,98 @@ If we compare the different fits, we can see that the Lorentzian model with diod
 While this is true for this particular dataset, no general statement can be made about the bias of fitting a Lorentzian rather than the hydrodynamically correct power spectrum.
 If low bias is desired, one should use the hydrodynamically correct model.
 On regular sensors, it is best to fit the hydrodynamically correct model with the diode model enabled.
+
+Active Calibration
+------------------
+
+Sometimes, more accuracy is required.
+One way to achieve this is with active force calibration.
+By providing a known perturbation to the bead, active calibration uses fewer assumptions than passive calibration.
+
+When performing passive calibration, we rely on knowledge of the drag coefficient to obtain a trap stiffness and positional calibration.
+This theoretical drag coefficient depends on parameters that are only known with limited precision.
+These parameters are the diameter of the bead and the viscosity.
+This viscosity in turn depends strongly on the local temperature around the bead, which is typically also poorly known.
+
+When performing active calibration, either the nanostage or trap is oscillated with a specific amplitude.
+This results in a visible driving peak at that particular frequency in the power spectrum.
+If we choose the windowing to include a full number of periods of the driving input, then the resulting driving peak will end up in a single data point in the frequency spectrum.
+
```

Comment:
Correct yes.

I think your phrasing is a lot more clear than mine. The only thing that I would change is that it's a peak at that frequency rather than that frequency itself. What do you think of the following:
```When the power spectrum is computed from an integer number of oscillations, the driving peak is visible at a single data point at :math:`f_{drive}`.```

In practice, what we actually do is chop the time series up into blocks that are an integer number of oscillations and average the spectra resulting from those (otherwise you get a bias from the fact that the errors on power spectral densities are not normally distributed).